### PR TITLE
Convert all bytebufs to bytebuffers

### PR DESCRIPTION
**Pull request by jianghaolu**

### CODE REVIEW

File: src/ClientModel/ClassType.cs [hunk 1]
```diff
@@ -40,7 +40,7 @@ public class ClassType : IType
         public static readonly ClassType RestProxy = new ClassType("com.azure.core.implementation", "RestProxy");
         public static readonly ClassType Validator = new ClassType("com.azure.core.implementation", "Validator");
         public static readonly ClassType Function = new ClassType("io.reactivex.functions", "Function");
-        public static readonly ClassType ByteBuf = new ClassType("io.netty.buffer", "ByteBuf");
+        public static readonly ClassType ByteBuffer = new ClassType("io.netty.buffer", "ByteBuffer");
```
Comment (alzimmermsft) [inline src/ClientModel/ClassType.cs]: Wrong import

Comment (jianghaolu) [reply src/ClientModel/ClassType.cs]: when find replace fails

Comment (jianghaolu) [reply src/ClientModel/ClassType.cs]: fixed

File: src/JvModel/PrimaryTypeJv.cs [hunk 2]
```diff
@@ -70,7 +70,7 @@ public string ModelTypeName
                         result = IsNullable ? "Long" : "long";
                         break;
                     case KnownPrimaryType.Stream:
-                        result = "Flowable<ByteBuffer>";
+                        result = "Flowable<ByteBufferfer>";
```
Comment (alzimmermsft) [inline src/JvModel/PrimaryTypeJv.cs]: ByteBuffer?

Comment (jianghaolu) [reply src/JvModel/PrimaryTypeJv.cs]: Hmm this is wrong in many ways. The path is somehow never hit. Fixing.

**Pull request merged**